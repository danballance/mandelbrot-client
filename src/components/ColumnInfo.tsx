import * as React from "react";
import { Grid } from 'semantic-ui-react'

export const ColumnInfo: React.FC = (): JSX.Element => (
    <Grid.Column only='computer' computer={8} textAlign="right">
        <div id="intro">
            <p>
        Click anywhere on the image below to re-centre on that locaton.<br/>
        Zoom in and out with the magnifying glass buttons to the right. 
            </p>
        </div>
    </Grid.Column>
)
