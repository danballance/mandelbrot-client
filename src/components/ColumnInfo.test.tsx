import React from 'react';
import { shallow } from 'enzyme'
import { ColumnInfo } from './ColumnInfo'

describe('ColumnInfo', (): void => {
    it('renders correctly', (): void => {
        const columnInfo = shallow(<ColumnInfo />);
        expect(columnInfo).toMatchSnapshot();
    })
});
