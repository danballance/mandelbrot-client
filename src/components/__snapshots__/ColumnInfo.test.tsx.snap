// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColumnInfo renders correctly 1`] = `ShallowWrapper {}`;
